components/Article.tsx

```diff
@@ -37,6 +37,7 @@ const Article: React.FunctionComponent<Props> = props => {
     const metaTags: MetaTags = {
         image: props.socialImage ?? props.image,
         description: props.description,
+        type: 'article',
```

Comment:
I also think different types of resources can have their own type in the future when we have more specific types. For now article in my opinion is ok. What do you think?